modules/react/icon/lib/Icon.tsx

```diff
@@ -18,6 +18,7 @@ export default class Icon extends React.Component<IconProps> {
 
     if (size) {
       iconStyles['& svg'] = {
+        ...((iconStyles['& svg'] as CSSObject) ?? {}),
```

Comment:
@NicholasBoll had to add this so Icon doesn't yeet any existing svg styles in the styles object